README.md

```diff
@@ -6,5 +6,7 @@ Usage is simple, on any request to `/` the request will result in a `200` respon
 
 A Docker image is available at: `quay.io/brancz/prometheus-example-app:v0.2.0`
 
+A [Deployment](manifests/deployment.yaml) and a [PodMonitor](manifests/pod-monitor.yaml) configuration are also available.
```

Comment:
Could we explain what PodMonitors are and link to some docs on them. Also mention you need prom-operator installed for this to work.

Reply:
On second thought, lets just add a section like this and explain and link to deployment and then add the info about PodMonitor:
````
## Deploying in a Kubernetes cluster
```

Reply:
Following your advice, I detailed the deployment part. Would you mind having another look at it? @lilic